contrib/guitarix/template.py

```diff
@@ -0,0 +1,45 @@
+pkgname = "guitarix"
+pkgver = "0.44.1"
+pkgrel = 0
+build_style = "waf"
+hostmakedepends = [
+    "binutils",
+    "faust",
+    "gettext",
+    "gperf",
+    "intltool",
+    "perl",
+    "pkgconf",
+    "python",
+    "sassc",
+]
+makedepends = [
+    "avahi-glib-devel",
+    "bluez-devel",
+    "boost-devel",
+    "eigen",
+    "fftw-devel",
+    "fonts-roboto-ttf",
+    "glib-devel",
+    "glibmm2.4-devel",
+    "gtk+3-devel",
+    "gtkmm3.0-devel",
+    "ladspa-sdk",
+    "libcurl-devel",
+    "liblo-devel",
+    "libsndfile-devel",
+    "lilv-devel",
+    "lrdf-devel",
+    "pipewire-jack-devel",
+    "zita-convolver-devel",
+    "zita-resampler-devel",
+]
+pkgdesc = "Virtual guitar amplifier"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "GPL-2.0-or-later"
+url = "https://guitarix.org"
+source = f"$(SOURCEFORGE_SITE)/guitarix/guitarix/guitarix2-{pkgver}.tar.xz"
+sha256 = "77e83d754f51ac38c5423f38eeb55de5b3e26128e60b511b02d2defcf36e6c18"
+tools = {"LD": "ld.bfd"}
```

Comment:
this seems like a huge hack

Reply:
(for fwupd-efi/gnu-efi this is because they do fancy bare elf things that are kinda hardcoded to binutils, but for a regular-ass userspace program i would really rather not see gnubinutils suddenly being used..)

Reply:
oh wow that's a lot of links, lemme see..

Reply:
uh based on https://github.com/brummer10/GxPlugins.lv2/issues/9#issuecomment-431634711 they use `-b -r` but those work in ld.lld?

Reply:
what error do you actually get without this?

Reply:
yeah, it's linking e.g. just a png into a .o. if you ass a real -m value (forgot the one for your computer) does it work?

Reply:
(you can test that by just running it in the build directory etc, no need to patch stuff)

Reply:
if not then yeah i guess lld just doesn't support whatever this is (never seen this before lol), very cursed

Reply:
hm ok so it does support it

maybe there's a better way to force it to output host elf, that doesn't involve an explicit -m..

or, maybe you can find out if/why this is even needed? i don't understand why it needs to start a -r object file with a png

Reply:
assuming the names are all magically correct (i thought there was weird mapping) this actually looks completely ok